src/protocols/vnc/display.c

```diff
@@ -182,22 +182,22 @@ static rfbBool guac_vnc_send_desktop_size(rfbClient* client, int width, int heig
     /* Get the Guacamole client data */
     guac_client* gc = rfbClientGetClientData(client, GUAC_VNC_CLIENT_KEY);
 
+    guac_client_log(gc, GUAC_LOG_TRACE,
+            "Current screen size is %ix%i; setting new size %ix%i\n",
+            rfbClientSwap16IfLE(client->screen.width),
+            rfbClientSwap16IfLE(client->screen.height),
+            width, height);
+
 #ifdef LIBVNC_CLIENT_HAS_SCREEN
-    /* Don't send an update if the sreen appears to be uninitialized. */
```

Comment:
I removed this because the screen size seems to be never initialized by the time that the owner attempts to set the initial screen size, so any initial screen resize will fail.

The screen size will be updated when a message comes back from the server specifying the updated size, but this in practice does not happen before the initial resize attempt.

In any case, removing this check fixes the initial resize issue, and doesn't seem to cause any problems that I can see.